### PR TITLE
[#162][Generate project with Mason] Migrate the rest of components generating step & clean up

### PR DESCRIPTION
**Pull request by manh-t**
- Solves #162

## What happened 👀

- To complete the Mason migration, we need to remove the old Python scripts and Makefile.
- Minor fix by replacing some hardcode app name in `Constants.py` with `app_name`

## Insight 📝

N/A

## Proof Of Work 📹

N/A

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,53 +0,0 @@
-.PHONY: help prepare-dev init test run
-
-# Python stuff
-VENV_NAME?=.venv
-PYTHON3 := $(shell command -v python3 2> /dev/null)
-VENV_ACTIVATE=$(VENV_NAME)/bin/activate
-PYTHON=$(VENV_NAME)/bin/python3
```
Comment (luongvo) [inline Makefile:7]: Should we clean up Python stuff in .gitignore? https://github.com/nimblehq/flutter-templates/blob/00b095d47ef275ec00d34faecd78efa4436fcb1b/.gitignore#L4-L6

**Review by luongvo — COMMENTED** [review]
The rest lgtm 💪

File: Makefile (hunk 1 above)
Comment (manh-t) [reply Makefile:7]: @luongvo nice catch. Updated in 72792b8

**Review by luongvo — APPROVED** [review]

**Review by markgravity — APPROVED** [review]
Nice works 💯

**Pull request merged**